apps/server/src/shared/repo/dashboard/dashboard.model.mapper.ts

```diff
@@ -89,7 +89,7 @@ export class DashboardModelMapper {
 
 	async mapDashboardToModel(entity: DashboardEntity): Promise<DashboardModelEntity> {
 		const existing = await this.em.findOne(DashboardModelEntity, entity.getId());
-		const modelEntity = existing || new DashboardModelEntity(entity.getId());
+		const modelEntity = existing || new DashboardModelEntity({ id: entity.getId(), user: entity.getUserId() });
```

Comment:
userId

Reply:
the property in the modelentity is actually called "user", in following with the other entities. The reason for that is that when handling the entity in code, the referenced entity can be populated directly into that property via a lazy loading mechanism.

Reply:
Aha! This is the point that I don't understand but still believe you.